CMakeLists.txt

```diff
@@ -12,35 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-project (sxg)
+project (sxg LANGUAGES C CXX)
 
 set (CMAKE_PROJECT_VERSION "0")
 set (CMAKE_PROJECT_VERSION_MAJOR "0")
 set (CMAKE_PROJECT_VERSION_MINOR "1")
 set (CMAKE_PROJECT_VERSION_PATCH "0")
 
+option (SKIP_TEST
+  "If true, build libsxg without library.  \
```

Comment:
Do you really mean "without library"??? (I'm confused.)

Reply:
sorry for typo.

Reply:
sorry for typo.